.travis.yml

```diff
@@ -6,36 +6,36 @@ branches:
 matrix:
   fast_finish: true
   include:
-      - os: linux
-        dist: xenial
-        sudo: required
+    - language: python
+      python: 3.6
+      dist: xenial
+      script: |
+        bash .ci/ci_check_commit.sh
 
-language: java
-jdk:
-  - openjdk8
-
-before_cache:
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
-
-cache:
-  directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
-
-before_install:
-  - gradle wrapper
-
-script: |
-  curl -LO https://raw.githubusercontent.com/FISCO-BCOS/FISCO-BCOS/master/tools/build_chain.sh && chmod u+x build_chain.sh
-  bash <(curl -s https://raw.githubusercontent.com/FISCO-BCOS/FISCO-BCOS/master/tools/ci/download_bin.sh) -b release-2.0.0
-  echo "127.0.0.1:4 agency1 1,2,3" > ipconf
-  ./build_chain.sh -e bin/fisco-bcos -f ipconf -p 30300,20200,8545 -v 2.0.0
-  ./nodes/127.0.0.1/start_all.sh
-  ./nodes/127.0.0.1/fisco-bcos -v
-  cp nodes/127.0.0.1/sdk/* src/integration-test/resources/
-  mv src/integration-test/resources/applicationContext-sample.xml src/integration-test/resources/applicationContext.xml
-  ./gradlew verifyGoogleJavaFormat
-  ./gradlew build
-  ./gradlew test
-  ./gradlew integrationTest
+    - language: java
+      jdk: openjdk8
+      os: linux
+      dist: xenial
+      sudo: required
+      before_cache:
+        - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+        - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
```

Comment:
just add a new python build, without modifying the config before

Reply:
add more ci now